### PR TITLE
VB-3824 integrate whereabouts with orchestration service

### PR DESCRIPTION
**Pull request by dhirajshettymoj**
Initial integration of whereabouts-api into Orchestration Service. Will add further code and tests when the story around whereabouts is further refined and added to the Sprint.

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/integration/mock/WhereaboutsApiMockServer.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          },
+        ),
+    )
+  }
+
+  private fun createJsonResponseBuilder(): ResponseDefinitionBuilder {
+    return aResponse().withHeader("Content-Type", MediaType.APPLICATION_JSON_VALUE)
+  }
+
+  private fun getJsonString(obj: Any): String {
```
Comment (adaviesMOJ) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/integration/mock/WhereaboutsApiMockServer.kt:37]: We duplicate this method a lot in the mock server tests. Is it possible to move it into a test-util type class?

Comment (dhirajshettymoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/integration/mock/WhereaboutsApiMockServer.kt:37]: Yes, good point. Will do that.

Comment (dhirajshettymoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/integration/mock/WhereaboutsApiMockServer.kt]: Added MockUtils class and moved both getJsonString and createJsonResponseBuilder as a companion object.

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/integration/mock/MockUtils.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder
+import com.github.tomakehurst.wiremock.client.WireMock
+import org.springframework.http.MediaType
+
+class MockUtils {
+  companion object {
+    private val objectMapper: ObjectMapper = ObjectMapper().registerModule(JavaTimeModule())
+      .setSerializationInclusion(JsonInclude.Include.NON_NULL)
+
+    fun getJsonString(obj: Any): String {
```
Comment (AledWEvans) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/integration/mock/MockUtils.kt:15]: You can put functions without classes, but no worries.

**Review by AledWEvans — APPROVED** [review]
LGTM

**Pull request merged**